File: backend/backend/web/api/sale/schema/sale.py
from typing import Optional

from backend.web.api.game.schema import GameInDB
from backend.web.api.platform.schema import PlatformInDB
from backend.web.api.sale.schema import SaleInDB
from backend.web.api.user.schema import UserInDB


class Sale(SaleInDB):
    game: GameInDB
    platform: PlatformInDB
    created_by_user: Optional[UserInDB] = None

File: frontend/src/client/models/Sale.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { GameInDB } from './GameInDB';
import type { PlatformInDB } from './PlatformInDB';
import type { UserInDB } from './UserInDB';

export type Sale = {
    id: string;
    amount: number;
    game: GameInDB;
    platform: PlatformInDB;
    created_by_user?: UserInDB;
};
